### ISSUE TITLE
 "cd: tmp: No such file or directory" when start sshcode

### ISSUE THREAD

**Issue by daiyk** [codercom/sshcode#48]
![Capture](https://user-images.githubusercontent.com/38419854/56567889-245c6900-65b6-11e9-9e95-31479060ae15.PNG)
I try to start sshcode under windows to connect server in Ubuntu, then when it install code-server I got the error message. I have checked server, /tmp and /tmp/code-server are there:
![Capture1](https://user-images.githubusercontent.com/38419854/56569435-3be92100-65b9-11e9-8377-6a536509425e.PNG)

I think it maybe the default login directory is /home/user? does someone know how to solve it or skip the step?

**Comment by coadler** [comment]
Are you running under WSL on windows? `sshcode` doesn't support windows natively

**Comment by daiyk** [comment]
> Are you running under WSL on windows? `sshcode` doesn't support windows natively

No, I run sshcode in build-in cmd, but the error is wired since it seems to have nothing to do with windows, and I think the error occurs during bash operations on server which is indeed linux......

**Comment by Naruto0** [comment]
Have the same error here, also on Windows. The command line is git-bash.

I think go is using windows path, which interprets slash as an backslash, but also over ssh so the Ubuntu reads it as escape character.

**Comment by daiyk** [comment]
> Have the same error here, also on Windows. The command line is git-bash.
> 
> I think go is using windows path, which interprets slash as an backslash, but also over ssh so the Ubuntu reads it as escape character.

I suspect that, the first 4 commands in fig that contains "slash" runs without problem on my server (it creates and downloads code-server). I don't familiar with go, but I think if someone change the call of 'cd tmp' to 'cd /tmp', it might work.

**Comment by roberthmiller** [comment]
/bin/bash should not exist on windows

**Comment by roberthmiller** [comment]
/tmp is not a dir in windows. I believe that tmp for apps in windows is within %APPDATA%.

**Comment by coadler** [comment]
I'm thinking it's probably an issue caused by the windows SSH command interpreting the bash script incorrectly since that script should be running on the remote host

**Comment by roberthmiller** [comment]
Could potentially be, I am not too familiar with windows.

**Comment by daiyk** [comment]
> /tmp is not a dir in windows. I believe that tmp for apps in windows is within %APPDATA%. I am pretty sure that windows support is eventually coming.

Actually, it succeed to execute that "wget" command to download code-server to /tmp in remote server, so I think the commands are run inside remote bash.
